website/content/en/docs/cloud-providers/AWS/launch-templates.md

```diff
@@ -0,0 +1,236 @@
+your subnet's network access control list (network ACL). 
+
+The security group must be associated with the virtual private cloud (VPC) of
+the EKS cluster.
+
+## Fields with Undefined Behavior
+
+These resources referenced by these fields are controlled by EKS/Karpenter, and not the launch template. 
+
+### Instance Type
+
+The instance type should not be specified in the launch template. Karpenter
+will determine the launch template at run time. 
+
+### Network Interfaces
+
+EKS will configure the network interfaces. Do not configure network instances
+in the launch template.
+
+## Creating the Launch Template
+
+Launch Templates may be created via the web console, the AWS CLI, or
+CloudFormation. 
+
+### CloudFormation
+
+The procedure, in summary, is to:
+1. [Create an AMI as described in the EC2 documentation.](https://docs.aws.amazon.com/vm-import/latest/userguide/vmimport-image-import.html)
+2. Write a EC2 Launch Template specification including the AMI.
+3. Push the specification to AWS with CloudFormation.
+4. Update the Provisioner CRD to specify the new Launch Template.
+
+An example yaml cloudformation definition of a launch template for Karpenter is
+provided below. 
+
+Cloudformation yaml is suited for the moderately high configuration density of
+launch templates, and creating the unusual InstanceProfile resource. 
+
+You must manually replace these values in the template:
+- SecurityGroupID
+  - list all security groups with `aws ec2 describe-security-groups`
+- Parameters in UserData
+
+```yaml
+AWSTemplateFormatVersion: '2010-09-09'
+Resources:
+  # create InstanceProfile wrapper on NodeRole
+  KarpenterNodeInstanceProfile:
+    Type: "AWS::IAM::InstanceProfile"
+    Properties:
+      InstanceProfileName: "KarpenterNodeInstanceProfile"
+      Path: "/"
+      Roles:
+        - Ref: "KarpenterNodeRole"
+  # create role with basic permissions for EKS node
+  KarpenterNodeRole:
+    Type: "AWS::IAM::Role"
+    Properties:
+      RoleName: "KarpenterNodeRole"
+      Path: /
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                !Sub "ec2.${AWS::URLSuffix}"
+            Action:
+              - "sts:AssumeRole"
+      ManagedPolicyArns:
+        - !Sub "arn:${AWS::Partition}:iam::aws:policy/AmazonEKSWorkerNodePolicy"
+        - !Sub "arn:${AWS::Partition}:iam::aws:policy/AmazonEKS_CNI_Policy"
+        - !Sub "arn:${AWS::Partition}:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly"
+        - !Sub "arn:${AWS::Partition}:iam::aws:policy/AmazonSSMManagedInstanceCore"
+  MyLaunchTemplate:
+    Type: AWS::EC2::LaunchTemplate
+    Properties: 
+      LaunchTemplateData: 
+        IamInstanceProfile:
+          # Get ARN of InstanceProfile defined above
+          Arn: !GetAtt
+            - KarpenterInstanceProfile
+            - Arn
+        ImageId: ami-074cce78125f09d61

```

Comment:
is this a public AMI? If so, do you need to specify the correct region for this to run? 

It might be better to use `<my-ami-id>` so the user knows to fill it in.

Reply:
As a personal preference, I like seeing what the actual value looks like. I have the list of values that must be replaced above.

Reply:
fair enough :)